actions/contacts/upsert.go

```diff
@@ -59,3 +59,18 @@ func (a *Action) upsert(c *gin.Context) {
 
 	c.JSON(http.StatusOK, response)
 }
+
+// upsertContact will add a new contact or modify an existing one.
```

Comment:
Same as for accept: use single method adjust description and use // @DeprecatedRouter

Reply:
I used @Deprecated with description about replacement.

Reply:
I used @Deprecated with description inside @Summary

Reply:
done (single method adjust description and use // @DeprecatedRouter)